src/main/scala/io/iohk/ethereum/jsonrpc/http/JsonRpcHttpServer.scala

```diff
@@ -31,10 +32,16 @@ class JsonRpcHttpServer(jsonRpcController: JsonRpcController, config: JsonRpcHtt
       }
       .result()
 
-  val route: Route =
+  val route: Route = {
     (pathEndOrSingleSlash & post & entity(as[JsonRpcRequest])) { request =>
       handleRequest(request)
+    } ~ post {
```

Comment:
If we wanted to log every rejection we could try something like this: http://stackoverflow.com/a/35165786 (stacking rejection handlers).

At any rate this cannot stay because it messes up our current rejection handler (try posting invalid JSON). Removing this is fine.